### PR TITLE
rdoc_alfred_workflow by dsg2157

### PR DESCRIPTION
**Pull request by dgorhe**
This is my GitHub submission for the Community Contribution project. Please let me know if there are any changes I should make or even any recommended changes.

### CODE REVIEW

File: rdoc_alfred_workflow.Rmd [hunk 1]
```diff
@@ -0,0 +1,52 @@
+```{r, include=FALSE}
```
Comment (Abhimanyu210100) [inline rdoc_alfred_workflow.Rmd:1]: The first line should be an '# <space> <Title>'. The second line should be blank and the third line should be your name

**Comment by Abhimanyu210100** [comment]
Can you please confirm that you received the comment and make appropriate changes @dgorhe

File: rdoc_alfred_workflow.Rmd (hunk 1 above)
Comment (dgorhe) [reply rdoc_alfred_workflow.Rmd]: Hi sorry for the late reply, I missed the notification on this pull request. I've made the changes and committed them. Do I need to submit a new pull request or will this one be merged?

**Comment by dgorhe** [comment]
I have updated the rmd file as you asked

**Comment by alexnsaunders** [comment]
@Abhimanyu210100 I haven't checked the underlying file but looks like this is ready to check and merge if all ok.

**Comment by Abhimanyu210100** [comment]
@dgorhe Thank you for your contribution 🎉

**Comment by Abhimanyu210100** [comment]
@alexnsaunders I have merged the file now

**Pull request merged**